packages/yext-function/mod.ts

```diff
@@ -6,8 +6,10 @@ import { default as render } from "./assets/renderer/templateRenderer.js";
  * the result.
  */
 export const PagesGenerator = async (data): Promise<Record<any, any>> => {
-  return await render({
-    document: data.streamOutput,
-    __meta: { manifest },
-  });
+  return await render(
+    {
+      document: data.streamOutput,
```

Comment:
Can you add the __meta.mode? It was missing so we may as well add it now. Should be "production" here.